index.html

```diff
@@ -3,8 +3,175 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <link href="styles.css" rel="stylesheet">
 </head>
 <body>
+<nav>
+    <ul>
+        <li><a href="xx" class="navbuttons">Case studies</a> </li>
+        <li><a href="xx" class="navbuttons">Services</a></li>
+        <li><a href="xx" class="navbuttons">Contact us</a></li>
+    </ul>
+</nav>
+<header class="outer-container">
+
+    <div class="inner-container">
+        <article class="top">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <h3> We are here to help you succeed.</h3>
+
+            <input type="button" class="button" value="let's work together!">
+        </article>
+        <img src="assets/images/squiggle_line_stripe.svg" id="squiggle" alt="squiggle" style="width:100px;height:75px">
```

Comment:
Als je gebruiktmaakt van een css file, dan zou ik persoonlijk geen inline styles gebruiken. Het is dan onoverzichtelijk waar welke css staat namelijk. Het is niet fout, maar het is beter voor de onderhoudbaarheid 😉

Reply:
Goed punt, ga ik meenemen voor de volgende!